### PR TITLE
cli: Support batch size on Linux.

### PR DESCRIPTION
**Pull request by tyll**
# Changes

**Which issue is resolved by this Pull Request:**
This solves one aspect of #379

**Description of your changes:**

Training speed can increase significantly with a higher batch size. Support to easily specify it on the command-line.

### CODE REVIEW

**Comment by tyll** [comment]
@russellb @hickeyma what's needed to get a review for this?

File: src/instructlab/lab.py [hunk 1]
```diff
@@ -885,6 +885,14 @@ def convert(self, value, param, ctx) -> "torch.device":
...
     help="model name to use in training",
 )
+@click.option(
+    "--batch-size",
+    "batch_size",
+    default=1,
+    help=(
+        "Batch size for training. Increases GPU usage."
+    )
+)
```
Comment (russellb) [inline src/instructlab/lab.py:895]: It looks like this is Linux-only? If so, could you update the text to indicate that?

Also, if you think about a user trying to use the tool, how should they decide if they should use this option? Is there text you could add to give any guidance?

Maybe the guidance should go in here?  https://github.com/instructlab/instructlab/blob/main/docs/gpu-acceleration.md

**Review by russellb — CHANGES_REQUESTED** [review]

**Comment by leseb** [comment]
@tyll do you mind addressing this review https://github.com/instructlab/instructlab/pull/997#pullrequestreview-2036839019 so we can move forward? Thanks!

**Review by leseb — COMMENTED** [review]
Apologies, we lost track of this and missed your latest update. If you're still interested in continuing this work, could you please rebase your PR? We’ll ensure it gets the attention it deserves. Thank you!

**Pull request closed without merging**